src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/mapping/ClaimMapper.java

```diff
@@ -48,11 +48,11 @@ private ClaimData toClaimData(final String input) {
     }
 
     private ResponseData toNullableResponseData(final String input) {
-        return toNullableEntity(input, ResponseData.class);
+        return input != null ? jsonMapper.fromJson(input, ResponseData.class) : null;
     }
 
     private CountyCourtJudgment toNullableCountyCourtJudgment(final String input) {
-        return toNullableEntity(input, CountyCourtJudgment.class);
+        return input != null ? jsonMapper.fromJson(input, CountyCourtJudgment.class) : null;
```

Comment:
I don't know, however I would have preferred
Optional.ofNullable(input).map(I-> jsonMapper.fromJson(i, CountyCourtJudgment.class)).orElse(null)

Reply:
reverted as it appearing after resolving conflicts.